inc/azure_uamqp_c/amqp_definitions_begin.h

```diff
@@ -0,0 +1,52 @@
+
+
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#ifndef AMQP_DEFINITIONS_BEGIN_H
```

Comment:
>P_DEFINITIONS [](start = 11, length = 13)

Consider:
It would be interesting maybe to have these generated types in a separate folder (probably very easy to change in the .cs that generates them.

And then we could improve the naming scheme a bit by having amqp_begin.h

So a user would include amqp_generated_types/amqp_begin.h for example.

I feel like it would be a cleaner presentation to the user.
This is a consider, but we could probably think a bit about it now. Anyway since most users are normally shielded by using the amqp_definitions.h main header, it should not be a big deal to change the naming scheme later also.